packages/web/src/js/constants.ts

```diff
@@ -1,9 +1,10 @@
-export const ARIA_EXPANDED_ATTRIBUTE = 'aria-expanded';
-export const ARIA_CONTROLS_ATTRIBUTE = 'aria-controls';
+export const ATTRIBUTE_ARIA_EXPANDED = 'aria-expanded';
+export const ATTRIBUTE_ARIA_CONTROLS = 'aria-controls';
+export const ATTRIBUTE_DATA_DISMISS = 'data-spirit-dismiss';
+export const ATTRIBUTE_DATA_TARGET = 'data-spirit-target';
+export const ATTRIBUTE_DATA_TOGGLE = 'data-spirit-toggle';
 
-export const NAME_DATA_TOGGLE = 'data-spirit-toggle';
-export const NAME_DATA_TARGET = 'data-spirit-target';
-
-export const CLASSNAME_EXPANDED = 'is-expanded';
-export const CLASSNAME_OPEN = 'is-open';
-export const CLASSNAME_TRANSITION = 'is-transitioning';
+export const CLASS_NAME_HIDDEN = 'is-hidden';
+export const CLASS_NAME_OPEN = 'is-open';
+export const CLASS_NAME_TRANSITIONING = 'is-transitioning';
+export const CLASS_NAME_VISIBLE = 'is-visible';
```

Comment:
you removed is-expanded, which could be used in the dropdown, where it is also declared.

Reply:
It could, but it wasn't 🙂.

I narrowed down the refactoring to:

1. reusing all useful constants,
2. deleting the unused ones,
3. creating a unified naming system.

Number 3 triggered the change in `Collapse.ts`, the only component that actually used the original constants.

All these (and many new!) constants can now be used in all our JS components. I just didn't do it yet because it's a fair amount of work which is better to do separately.